cmd/activator/main.go

```diff
@@ -145,8 +145,19 @@ func main() {
 		logger.Fatalw("Error building serving clientset", zap.Error(err))
 	}
 
-	if err := version.CheckMinimumVersion(kubeClient.Discovery()); err != nil {
-		logger.Fatalf("Version check failed: %v", err)
+	// We sometimes startup faster than we can reach kube-api. Poll on failure to prevent us terminating
+	// our pod
+	startTime := time.Now()
+	for {
```

Comment:
I think it might be nicer to have a fixed number of tries rather than a fixed time. If the first request takes awhile to return failure you many only try once. Why not do 5 tries with 1 second sleep between?